packages/zowe-explorer-api/src/extend/IRegisterClient.ts

```diff
@@ -110,6 +110,16 @@ export interface IRegisterClient {
      */
     onProfilesUpdate?: vscode.Event<Validation.EventType>;
 
+    /**
+     * Define events that fire whenever credentials are updated on the client.
+     */
+    onVaultUpdate?: vscode.Event<Validation.EventType>;
+
+    /**
+     * Define events that fire whenever the credential manager is updated.
+     */
+    onCredMgrsUpdate?: vscode.Event<Validation.EventType>;
```

Comment:
As we currently support only one CredentialManager to be used at a time, should this be singular - `onCredMgrUpdate`? I believe that would more closely match the event name defined by Imperative 😋